tests/schema/__snapshots__/computedInputs.test.ts.snap

```diff
@@ -321,15 +321,15 @@ interface NexusPrismaInputs {
   ordering: 'id' | 'name'
 }
     nesteds: {
-  filtering: 'id' | 'name' | 'AND' | 'OR' | 'NOT' | 'user'
-  ordering: 'id' | 'name'
+  filtering: 'id' | 'name' | 'userId' | 'AND' | 'OR' | 'NOT' | 'user'
+  ordering: 'id' | 'name' | 'userId'
 }
 
   },
     User: {
     nested: {
-  filtering: 'id' | 'name' | 'AND' | 'OR' | 'NOT' | 'user'
-  ordering: 'id' | 'name'
+  filtering: 'id' | 'name' | 'userId' | 'AND' | 'OR' | 'NOT' | 'user'
+  ordering: 'id' | 'name' | 'userId'
```

Comment:
Hm, this looks undesirable?

Reply:
But I don't think this is critical. Maybe I'm wrong but just looks like additive noise to the GraphQL schema.

Reply:
Yeah, nothing critical, it's just because we're not filtering the additional fields now required to create relations!